packages/core/src/lib/markdown-it/highlight/HighlightRule.ts

```diff
@@ -61,56 +66,33 @@ export class HighlightRule {
     return atLineNumber;
   }
 
-  applyHighlight(line: string, lineNumber: number) {
-    // Applied rule is the first component until deduced otherwise
```

Comment:
Is this comment still relevant on line 73? if so can we move it down and keep it?

Reply:
I don't think it is relevant cos rather than first comp it is based on whole line > whole text > partial

Reply:
hmm, as in `let [appliedRule] = this.ruleComponents;`, isn't this
> Applied rule is the first component until deduced otherwise

?